sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Subscribe.java

```diff
@@ -68,10 +68,13 @@
      * kalix.javasdk.eventsourcedentity.EventSourcedEntity EventSourcedEntity}.
      */
     Class<? extends kalix.javasdk.eventsourcedentity.EventSourcedEntity<?>> value();
+
+    /** This option is only available for classes. Using it in a method doesn't change the default. */
```

Comment:
this docs is only saying that it won't work in methods, but it doesn't say what it means. 

We should explain what `ignoreUnknown` means here. 

Once we explain it, we can then say that it's only effective when used at type level and that the option is ignore on method level. Better not to say "doesn't change the default", but simply say is not applicable.

Reply:
reworded and added functionality description